### ISSUE TITLE
No container image

### ISSUE THREAD

**Issue by mjschmidt** [Mirantis/virtlet#896]
kubectl create -f https://raw.githubusercontent.com/Mirantis/virtlet/master/examples/cirros-vm.yaml

There is not image for this docker file?
I am following the example found [here](https://docs.virtlet.cloud/user-guide/real-cluster/)

**Comment by mjschmidt** [comment]
I also tried changing this image out for cirros:latest. I was unable to attach to the pod, but this may be unrelated as I did not yet install the CRI Proxy to my cluster. I just wanted to test out the deployment.

**Comment by pigmej** [comment]
It's not docker container. it's vm image. If your system tried to fetch container image then probably cri-proxy is missconfigured.

**Comment by jellonek** [comment]
There was also a comment: "I also tried changing this image out for cirros:latest. I was unable to attach to the pod, but this may be unrelated as I did not yet install the CRI Proxy to my cluster. I just wanted to test out the deployment." (which looks like was removed) which clearly says about missing cri proxy.

@mjschmidt have you added it and reconfigured kubelet to use cri proxy? If so - does this issue is still valid?
